pkg/kubelet/pod_workers.go

```diff
@@ -665,6 +675,13 @@ func (p *podWorkers) UpdatePod(options UpdatePodOptions) {
 		podUpdates = make(chan podWork, 1)
 		p.podUpdates[uid] = podUpdates
 
+		if kubetypes.IsStaticPod(pod) {
```

Comment:
Since this chunk is about ensuring that static pods start in FIFO order, please add a comment to this chunk like:

```
// ensure that static pods start in the order they are received by UpdatePod
```